src/dotnet/APIView/APIView/Model/CodeDiagnosticLevel.cs

```diff
@@ -9,4 +9,4 @@ public enum CodeDiagnosticLevel
         Warning = 2,
         Error = 3
     }
-}
\ No newline at end of file
+}
```

Comment:
Looks like this sneaked in from another work.

Reply:
Better to accept the trailing newline so we don't continuously get this file changed whenever someone's editor opens it.